catherine_feijoo/class2/compare.rb

```diff
@@ -0,0 +1,12 @@
+puts "plz enter an int. to end enter -1."
+
+user_num = 0
+
+while user_num != -1
+  user_num = gets.chomp.to_i
+  if (user_num % 2) == 0
+    puts "even"
+  else
+    puts "odd"
+  end
```

Comment:
I love the loop! Maybe on the line before the loop ends, you can prompt the user to enter another integer?